ci/container/test_component.sh

```diff
@@ -67,7 +67,7 @@ env | grep SNOWFLAKE_ | grep -v PASS
 echo "[INFO] Starting hang_webserver.py 12345"
 python3 $THIS_DIR/hang_webserver.py 12345 > hang_webserver.out 2>&1 &
 
-if [[ "$SHOULD_GENERATE_COVERAGE_REPORT" -eq "1" && "$CLOUD_PROVIDER" == "AWS" ]];
+if [[ "$SHOULD_GENERATE_COVERAGE_REPORT" -eq "1" ]];
```

Comment:
Does it work? I believe that `-eq` is used to compare numbers, for strings `==` should be used?

Reply:
Yes, it works - for Mac we are gathering the coverage in tests (https://github.com/snowflakedb/snowflake-connector-nodejs/actions/runs/5830408639/job/15812373008?pr=612) and for linux is not gathered (https://github.com/snowflakedb/snowflake-connector-nodejs/actions/runs/5830408639/job/15812372847?pr=612) - in both links look at the `Tests` step

Reply:
Can we change it to be better POSIX ready?

Reply:
replaced with `==`